### PR TITLE
Restrict workspace/symbol results to non-dependency symbols

### PR DESCRIPTION
**Pull request by felixfbecker**
This improves performance a lot and matches what other language servers do

### CODE REVIEW

**Comment by jens1o** [comment]
Could you explain how you improve performance with a simple variable rename and new docs, please?

**Comment by felixfbecker** [comment]
As the title says, this restricts the search to only non-dependency symbols

**Comment by jens1o** [comment]
Okay, thanks. When will you release the new version?

File: src/Server/Workspace.php [hunk 1]
```diff
@@ -84,11 +84,11 @@ public function symbol(string $query): Promise
     {
         return coroutine(function () use ($query) {
             // Wait until indexing for definitions finished
-            if (!$this->index->isStaticComplete()) {
-                yield waitForEvent($this->index, 'static-complete');
+            if (!$this->sourceIndex->isStaticComplete()) {
```
Comment (staabm) [inline src/Server/Workspace.php]: @jens1o its not just a rename. in some spots its now using `sourceIndex` in others `projectIndex`

Comment (jens1o) [inline src/Server/Workspace.php]: Yeah, I looked over this, but after he said it I got it.

**Pull request merged**